arpsync/watcher.go

```diff
@@ -77,7 +77,7 @@ func buildContainersMap(containers []metadata.Container,
 	for index, aContainer := range containers {
 		if !(aContainer.PrimaryIp != "" &&
 			aContainer.PrimaryMacAddress != "" &&
-			(aContainer.State == "running" || aContainer.State == "starting") &&
+			(aContainer.State == "running" || aContainer.State == "starting" || aContainer.State == "stopping") &&
```

Comment:
Nit: may be this container state comparison can be moved to a separate util call? something like "isRunningContainer" returning boolean. This way it will be maintained in a single place